app/controllers/home_controller.rb

```diff
@@ -2,7 +2,7 @@ class HomeController < ApplicationController
   def index
     @match_date = params[:match_date] ? DateTime.new(params[:match_date][:year].to_i, params[:match_date][:month].to_i, params[:match_date][:day].to_i) : DateTime.now
 
-    @matches = Match.played_during_month(@match_date)
+    @matches = Match.where(event_id: nil).played_during_month(@match_date)
```

Comment:
For now, we are treating "season" matches as completely separate from "event" matches. Until we get a more formalized definition of how we want to handle these... this will at least allow us to calculate separate rankings and elos